docs/source/releases/5.2.x.rst

```diff
@@ -12,6 +12,8 @@ New features
 
 - Added :class:`~.fields.EmbeddedModelArrayField` for storing data as an array
   of model instances.
+- Added support for ``exact``, ``overlap``, and ``len`` lookups on
```

Comment:
Since this will be in the same release as the initial introduction of the field, a separate note isn't needed.